package.json

```diff
@@ -3,7 +3,7 @@
   "engines": {
     "node": ">=16"
   },
-  "version": "8.1.152",
+  "version": "8.1.153",
```

Comment:
Bump is only for the 8.1 branch. The 8.2 will be released in another PR.

Reply:
All Artesca versions in prod are currently using it: I believe that we'll want to have the fixes around the ghost issue for previous versions as well for some customers... So we should add the changes in 8.1.